examples/paths.rb

```diff
@@ -51,6 +51,13 @@ def path_to(page_name)
 
   private
 
+  def path_to_show_or_edit(action_prose, model_prose, identifier)
+    model = model_prose.classify.constantize
```

Comment:
I think, you should use `model_prose_to_class` (defined below) here. This will replace whitespaces with underscores before constantizing.

Reply:
Good point, thanks!